manifests/dev/rbac.authorization.k8s.yaml.in

```diff
@@ -125,6 +126,21 @@ rules:
     verbs:
       - get
       - list
+      - watch
+  - apiGroups:
+      - ''
+    resources:
+      - limitranges
```

Comment:
does virt-controller no longer need access to limitranges now? If so it seems like we could remove that permission from the virt-controller ClusterRole

Reply:
done